src/bci_build/package/nginx/README.md.j2

```diff
@@ -0,0 +1,103 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+Nginx (pronounced "engine-x") is an open-source reverse proxy server for HTTP, HTTPS, SMTP, POP3, and IMAP protocols, as well as a load balancer, HTTP cache, and a web server (origin server).
+
+## Usage
+
+By default, the image launches Nginx with the same configuration that comes with SUSE Linux Enterprise Server.
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 {{ image.pretty_reference }}
+```
+
+Or:
+
+```ShellSession
+$ podman run -it --rm -p 8080:80 -v /path/to/html/:/srv/www/htdocs/:Z {{ image.pretty_reference }}
+```
+
+**Note:** The directory `/srv/www/htdocs/` is the root directory used by the default server. Additional servers can use any other path.
+
+Once the container is running, access `http://localhost:8080` or `http://host-ip:8080` in your browser.
+
+## Environment variables
+
+### NGINX_ENTRYPOINT_QUIET_LOGS
+
+This optional environment variable controls the logging during container startup. Set the value to `1` to silence logs.
+
+### NGINX_ENTRYPOINT_WORKER_PROCESSES_AUTOTUNE
+
+This optional environment variable enables a script to autotune the number of worker processes. Set the value to `1` to enable autotune of the worker process parameter.
+
+## Using templates
+
+Out of the box, nginx doesn't support environment variables inside configuration blocks, but this image has a script that can extract environment variables before nginx starts.
+
+By default, this script reads template files in `/etc/nginx/templates/*.template` and outputs the result of executing `envsubst` to `/etc/nginx/conf.d`.
+
+If you create a file named `/etc/nginx/templates/default.conf.template`, which contains a variable reference like this:
+
+```
+listen       ${NGINX_PORT};
+```
+
+The script will output it to `/etc/nginx/conf.d/default.conf` like this:
+
+```
+listen       80;
+```
+
+The use of templates allows you to set arbitrary environment variables that can be replaced inside the container, such as `-e NGINX_PORT=80`.
+
+### NGINX_ENVSUBST_TEMPLATE_DIR
+
+This optional environment variable changes the directory that contains template files (default: `/etc/nginx/templates`).
+
+**Note:** The script will ignore template processing if this directory doesn't exist
+
+### NGINX_ENVSUBST_OUTPUT_DIR
+
+This optional environment variable changes the directory where the result of executing `envsubst` on templates is output (default: `/etc/nginx/conf.d`).
+
+The output filename is the template filename with the suffix removed.
```

Comment:
Does this actually work on SLES? I guess you'd have to adjust the load path for nginx config files as well, right?

Reply:
Yes, probably needs a note here mentioning that if you change this path, you must change nginx.conf otherwise it will still load the old path.